gradle.properties

```diff
@@ -1,14 +1,162 @@
-modGroup=eutros
-modBaseName=multiblocktweaker
-phase=stable-gtceu
-major=1
-minor=8
-patch=2
-
-forge_ver=1.12.2-14.23.5.2847
-gt_ver=1.17.0.764
-ct_ver=1.12-4.1.20.655
-jei_ver=4.15.0.293
-ctm_ver=MC1.12.2-1.0.2.31
-ccl_ver=3.2.3.357
-        
+modName = Multiblock Tweaker
+
+# This is a case-sensitive string to identify your mod. Convention is to use lower case.
+modId = multiblocktweaker
+
+modGroup = eutros
+
+# Version of your mod.
+# This field can be left empty if you want your mod's version to be determined by the latest git tag instead.
+modVersion = 1.12.2-stable-gtceu-1.8.2
+
+# Whether to use the old jar naming structure (modid-mcversion-version) instead of the new version (modid-version)
+includeMCVersionJar = false
```

Comment:
Could enable this setting to not need to include 1.12.2 in the tracked internal version (not sure how it was done before)

Reply:
Since the 1.12.2 would not show up in game, leaving as is to keep the name the same in game